### PR TITLE
Feature/clips improvements

### PR DESCRIPTION
**Pull request by glassresistor**
Continues  #1049 

### What does this do and why?

This added error handling and more purged views.

### Testing & Checks

What does this PR include?
- [ ] new dependencies
- [x] migrations
- [x] data migrations
- [ ] celery tasks
- [ ] changes to environment variables or local settings
- [ ] configuration changes for 3rd party services (Travis, Heroku, AWS, Github, etc) 
- [ ] visual or style changes

Are there any human checks that should be used to verify these changes, such as user acceptance tests (UAT) or manual deployment checks?

Its not a blocker but people can check all of the new views work.

Any integration or unit tests?

No

### CODE REVIEW

File: intake/migrations/0063_purgedapplication_purgedstatusupdate.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    operations = [
+        migrations.CreateModel(
+            name='PurgedApplication',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+            ],
+            options={
+                'managed': False,
+                'db_table': 'purged"."intake_application',
```
Comment (bengolder) [inline intake/migrations/0063_purgedapplication_purgedstatusupdate.py:22]: quotation seems weird, will this be quoted properly in the end?

**Comment by bengolder** [comment]
This looks like it does some unusual quotation stuff in the db names. Has this been tested to make sure it won't break?
```python
# in migration
'db_table': 'purged"."intake_application',
```
```python
# in model
db_table = 'purged\".\"intake_application'
```

There are some things I don't understand:
- Why are there models, as opposed to just the SQL to make the views?
- Why are the models in each of the corresponding model files? Wouldn't it make more sense to include them in a separate app or in clips?
- All of the "Purged" models have the same TODO listed in their docstring. Is this intentional? If so, can we pull this into an issue instead of a TODO comment in the code?

**Comment by glassresistor** [comment]
This looks like it does some unusual quotation stuff in the db names. Has this been tested to make sure it won't break?

Yes it looks strange but thats the only way to do it have it not break on write and is considered the right way to do it.  When django make the sql what ends up happening is schema"."table_name gets wrapped in quotes and becomes "schema"."table_name".  Since theres are views it doesn't matter but its good to be consistent.

There are some things I don't understand:
- Why are there models, as opposed to just the SQL to make the views?
  The models help in afew ways.  First they work as a placeholder and facilitate the automatic creation of the migration file(unneeded but tidy).  We can also go in later and add the fields if we want(or subclass) and then access the PII cleaned views like normal Django models.  We don't need that now but its easy to do and having a placeholder for unmanagered tables is a best practice.
- Why are the models in each of the corresponding model files? Wouldn't it make more sense to include them in a separate app or in clips?
  I felt it makes the most sense to have each beside the model its related to.  It makes it easier to keep things organized.  It also will make it easier to pull in the fields to the Purged models if we want to use them in Django later.
- All of the "Purged" models have the same TODO listed in their docstring. Is this intentional? If so, can we pull this into an issue instead of a TODO comment in the code?
  That was an accident thanks I'll remove all but the original.  TODO is traditionally used to let developers know what future work could or should be be done somewhere.  I didn't add a ticket because I don't think its something we have a need to do but if someone comes along its good to have there.  In this case I'll remove the todo and just leave a note that it can be done.

**Comment by glassresistor** [comment]
Ok the TODOs from the docstrings were removed and the migrations were properly referenced.

**Pull request merged**